core/cmd/remote_client_test.go

```diff
@@ -443,6 +443,16 @@ func TestClient_ChangePassword(t *testing.T) {
 	err = otherClient.RemoteLogin(c)
 	require.NoError(t, err)
 
+	client.ChangePasswordPrompter = cltest.MockChangePasswordPrompter{
+		UpdatePasswordRequest: web.UpdatePasswordRequest{
+			OldPassword: testutils.Password,
+			NewPassword: "12345",
+		},
+	}
+	err = client.ChangePassword(cli.NewContext(nil, nil, nil))
+	require.Error(t, err)
+	assert.Contains(t, err.Error(), "Expected password complexity")
```

Comment:
Could simplify these a bit with the `ErrorContains` helper:
```suggestion
	assert.ErrorContains(t, err, "Expected password complexity")
```

Reply:
Fixed in b87aac9